index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <section class="header-section">
+        <div class="header-container">
+          <div class="title">
+            <img class="logo" src="image/logo.png" alt="logo" />
+            <a href="/">
+              <img class="pandamarket-logo" src="image/pandamarket.png" alt="pandamarket" />
+            </a>
+          </div>
+          <a class="login-button" href="/login.html">로그인</a>
+        </div>
+      </section>
+      <section class="content-section">
+        <div class="main-banner">
+          <div class="banner-text">
+            <h2>
+              일상의 모든 물건을<br />
+              거래해 보세요
+            </h2>
+            <a class="browse-button" href="/items.html">구경하러 가기</a>
+          </div>
+          <img class="panda-image" src="image/panda.png" alt="" />
+        </div>
+        <div class="container">
+          <div class="category">
+            <img class="hotitem-image" src="image/hotitem.png" alt="hotitem" />
+            <div class="category-detail">
+              <span>Hot item</span>
+              <div class="flex-text">
+                <h2>인기 상품을 <br />확인해 보세요</h2>
+              </div>
```

Comment:
여기 `div`는 없어도 될 것 같네요 ~ 
`text-align` 속성으로 컨트롤해도 될 것 같습니다. 
```suggestion
                <h2>인기 상품을 <br />확인해 보세요</h2>
```